tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -36,12 +36,13 @@ test.describe( 'Price Benchmark Page', () => {
 		await page.close();
 	} );
 
+	test.beforeEach( async () => {
```

Comment:
@ankitrox Why do we need to go to the price benchmark page before each test? 🤔

Reply:
Removed it from here, but we will need this in `Price Benchmark Suggestions Functionality` because we have variable suggestions for each test case.